File: Java/Burbuja.java
import java.util.Random;
import java.util.Arrays;
public class Burbuja {

    public static void main(String[] args) {
        Random r1 = new Random();
        int[] arr = new int[10];    

        for (int i = 0; i < 10;i++){
            int n2 = r1.nextInt(100);
            arr[i] = n2;
        }
        
        System.out.println("El arreglo desordenado es: ");
        System.out.println(Arrays.toString(arr));

        for (int i = 0; i < arr.length; i++){
            for (int j = 0; j < arr.length-i-1; j++){
                if (arr[j] > arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }

            }
        }

        System.out.println("El arreglo ordenado es: ");
        System.out.println(Arrays.toString(arr));

    }


    
}

File: C#/Burbuja.cs

class Program
{
    static void Main()
    {
        Random rnd = new Random();
        int[] arr = new int[10];

        for (int i = 0; i < arr.Length; i++)
        {
            int numero = rnd.Next(1, 100);
            arr[i] = numero;
        }

        Console.WriteLine("El arreglo desordenado es: ");
        Array.ForEach(arr, x => Console.Write(x + " "));  

        for (int i = 0;i < arr.Length; i++)
        {
            for (int j = 0; j<arr.Length-i-1; j++)
            {
                if (arr[j] > arr[j + 1])
                {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;  
                }
            }
        }
        Console.WriteLine("");
        Console.WriteLine("El arreglo ordenado es: ");
        Array.ForEach(arr, x => Console.Write(x + " "));

    }



}
